### ISSUE TITLE
RFE: Add Tower-managed PostgreSQL replication

### ISSUE THREAD

**Issue by tesseract-rh** [ansible/awx#7844]
##### ISSUE TYPE
 - Feature Idea

##### SUMMARY
[Ansible Tower Administration Guide v3.7.2](https://docs.ansible.com/ansible-tower/latest/html/administration/clustering.html#setup-considerations) (i.e. `latest` at the moment of writing):
> PostgreSQL is still a standalone instance and is not clustered. Tower does not manage replica configuration or database failover (if the user configures standby replicas).

This is a request for adding PostgreSQL replication management to Tower.

##### Justification: 

PostgreSQL replication is a must for eliminating SPOF (Single Point Of Failure). However, currently we can have that feature available only if the DB is _not_ managed by Ansible Tower.

In a business context that adds to unnecessary complication of both setting up and upgrading Tower: usually there's a separate team that takes care of the DB setup, and therefore upgrades need to be carefully examined for additional requirements/dependencies.

In short, the upgrade cadence becomes much more cumbersome and prone to mistakes.

##### Downstream links:
[Is Database Replication Supported with an Ansible Tower subscription?](https://access.redhat.com/solutions/3682951) 
[Red Hat Ansible Tower Database Scope of Coverage](https://access.redhat.com/articles/4010491)
